lib/Util/PathNormalizer.php

```diff
@@ -0,0 +1,37 @@
+<?php
+
+namespace PhpBench\Util;
+
+use Webmozart\PathUtil\Path;
+
+class PathNormalizer
+{
+    /**
+     * @param string[] $paths
+     *
+     * @return string[]
+     */
+    public static function normalizePaths(string $baseDir, array $paths): array
+    {
+        return array_merge(...array_map(static function (string $path) use ($baseDir) {
+            $path = Path::isAbsolute($path) ? $path : Path::join([$baseDir, $path]);
+
+            if (self::isGlob($path)) {
+                $globPaths = glob($path, GLOB_NOSORT);
+
+                if (empty($globPaths)) {
+                    return [];
+                }
+
+                return $globPaths;
+            }
+
+            return [ $path ];
+        }, $paths));
+    }
```

Comment:
@ricardoboss i extracted the logic to a static funtion and added dedicated tests. The original version didn't work with absolute paths